students/WesWang/lesson4/mailroom_part2.py

```diff
@@ -0,0 +1,83 @@
+#=========================================================================================
+#Name: Mailroom
+#Author: Wesley Wang
+#Date: 4/23/2018
+#=========================================================================================
+
+"""
+Mailroom Part2
+"""
+import sys
+
+donors = {"Donor A":[100000, 40000]
+        , "Donor B":[1000]
+        , "Donor C":[1000,1000,1000]
+        , "Donor D":[5000,10000,5000]}
+
+
+def thank_letter(donor, amount):
+    print("-"*80)
+    print(f"Dear {donor},\n\n\tWe have received you donation of ${amount:,.2f}. We greatly appreciate your generous contribution. This donation is meaningful and crucial to our organization's misson.\n\n\nSincerely,\n\nWesley Wang\n")
+    print("-"*80)
+
+def thank_you():
+    # Ask for user input
+    response = input("Please enter donor's full name, or type 'list' to check all donors: ").title()
+    if response.lower() == "list":
+        print("-"*20 + "\nList of Donors:\n" + "-"*20)
+        for donor in donors.keys():
+            print(donor)
+        print("-"*20)
+        thank_you()
+    amount = int(input("Please enter donation amount for " + response + ":"))
```

Comment:
Casing the amount to an int() makes it so that if someone donates "57.34" throw an error. So we only allow full dollar amounts, which is fine if that's what you want, just be aware that you can't accept non-whole dollar donations